src/snapshot/filesaver.js

```diff
@@ -53,7 +53,11 @@ var fileSaver = function(url, name) {
 
         // IE 10+ (native saveAs)
         if(typeof navigator !== 'undefined' && navigator.msSaveBlob) {
-            navigator.msSaveBlob(new Blob([url]), name);
+            // At this point we are only dealing with a SVG encoded as
+            // a data URL (since IE only supports SVG)
+            var encoded = url.split(/^data:image\/svg\+xml,/)[1];
```

Comment:
I wonder what's better (i.e. faster) 

```js
url.split(/^data:image\/svg\+xml,/)[1];

// or as in plot_api/to_image.js
url.replace(/^data:image\/svg\+xml,/, '');
```

Reply:
I don't see perf as too important in this block. An advantage (at least it looks like an advantage to me...) of the current one is that if `url` does NOT start with the expected `'data:image/svg+xml,'` it will throw an error, rather than saving garbage, which seems easier to investigate.

Reply:
> it will throw an error, rather than saving garbage,

Good point. :+1: